### ISSUE TITLE
Proposal: Allow ctrl + enter to send the post

### ISSUE THREAD

**Issue by protodrew** [cblgh/cerca#32]
I do it a ton unconsciously because it exists in mastodon. Might be nice but Idk how complex it might be.

**Comment by Thomasorus** [comment]
Since it’s a progressive enhancement that may be doable without affecting the rest of the forum, what do you think @cblgh ?

**Comment by cblgh** [comment]
@Thomasorus yes! this is ofc something we can add :) rn i'm just ambiently mulling about the strategy for progressive enhancements, such that they will improve things without increasing the maintenance burden (e.g. needing to be updated or breaking down the line)

**Comment by milofultz** [comment]
heyo, I'm bored at christmas and wondering if I can kick around with this and see if it works? I haven't set up a dev environment yet, so I'll see how far I get with that first. But this seems like something you could do by creating a template HTML file with a `script` that makes an event listener that focuses on `textarea` elements within `form` elements, and then including that at the bottom of the `thread` and `new-thread` templates.
